cc_library/src/scripts/start_door.py

```diff
@@ -0,0 +1,59 @@
+import os
+import time
+
+from src.device.app import App
+
+try:
+    import RPi.GPIO as GPIO
+except (RuntimeError, ModuleNotFoundError):
+    from fake_rpi.RPi import GPIO
+
+
+class Door:
+    def __init__(self):
+        self.door = 17
+        GPIO.setup(self.door, GPIO.OUT)
+
+    def incoming_instruction(self, data):
+        """
+        Set here the mapping from messages to methods.
+        """
+        instruction = data.get("instruction")
+        if instruction == "test":
+            self.test()
+        elif instruction == "turn off":
+            self.turn_off(data)
+        elif instruction == "turn on":
+            self.turn_on(data),
+
+    def turn_off(self, data):
+        # door = getattr(self, data.get("door"))
```

Comment:
if this is no longer used, removed commented code

Reply:
It is currently hardcoded but if we do not want that later we will need that code